### PR TITLE
REF only include host/build for things we build

### PR DESCRIPTION
**Pull request by beckermr**
This is a follow-on to clip the graph more.

### CODE REVIEW

File: conda_forge_tick/migrators/migration_yaml.py [hunk 1]
```diff
@@ -492,9 +492,10 @@ def create_rebuild_graph(
         )
         # get host/build, run and test and launder them through outputs
         # this should fix outputs related issues (eg gdal)
-        all_reqs = (host or build) | requirements.get("run", set())
+        all_reqs = requirements.get("run", set())
```
Comment (isuruf) [inline conda_forge_tick/migrators/migration_yaml.py:495]: This is populated with `run_exports` from `host` ?

Comment (beckermr) [reply conda_forge_tick/migrators/migration_yaml.py:495]: In an approximate way, yes: https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/make_graph.py#L172

The approximation made in the code in the link is not different from keeping host/build sections. Thus I think this should be ok.

Comment (beckermr) [reply conda_forge_tick/migrators/migration_yaml.py:495]: The code there is not perfect so we might hold off on this change for a while.

Comment (beckermr) [reply conda_forge_tick/migrators/migration_yaml.py:495]: Actually no, let's ship this now and see what happens.

**Pull request merged**